docs/src/styles/docs/base.scss

```diff
@@ -38,8 +38,6 @@ html {
 
 // Remove this when this gets fixed in the UI package
 * {
-  box-sizing: border-box;
```

Comment:
This was only affecting the docs site. Removing as it's redundant with the new CSS rule below.